build.sbt

```diff
@@ -145,7 +145,7 @@ val forkSettings = Seq(
 
 val akkaV = "2.5.21" //same version as in Flink
 val flinkV = "1.9.1"
-val avroV = "1.8.2" // should be the same as in flink-avro
+val avroV = "1.9.1" // should be the same as in flink-avro
```

Comment:
what about this comment?

Reply:
`"org.apache.flink" % "flink-avro" % flinkV,` - right now is the same..

Reply:
You're right. A will rollback this.